### PR TITLE
Add missing debugs

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [inline js/flightlog_fields_presenter.js:1850]: We should set the display name of each debug field here, so that it shows up in the log explorer.

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [inline js/flightlog_fields_presenter.js:1118]: These are new debugs.  They should be included in the initial debug list, not here.  Conditional server checks are only required if an existing debug has field names changed in a later firmware version.

**Comment by ctzsnooze** [comment]
Could I suggest adding these graph settings into `graph_config.js`, for Mag_Calib and Mag_Task_rate:
```
                   case 'MAG_CALIB':
                        switch (fieldName) {
                            case 'debug[0]': // X
                            case 'debug[1]': // Y
                            case 'debug[2]': // Z
                            case 'debug[3]': // Field
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 2000,
                                    outputRange: 1.0,
                                };
                            case 'debug[4]': // X Cal
                            case 'debug[5]': // Y Cal
                            case 'debug[6]': // Z Cal
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 500,
                                    outputRange: 1.0,
                                };
                            case 'debug[7]': // Lambda
                                return {
                                    offset: -2000,
                                    power: 1.0,
                                    inputRange: 2000,
                                    outputRange: 1.0,
                                };
                            default:
                                return getCurveForMinMaxFields(fieldName);
                            }
                    case 'MAG_TASK_RATE':
                        switch (fieldName) {
                            case 'debug[0]': // Task Rate
                            case 'debug[1]': // Data Rate
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 1000,
                                    outputRange: 1.0,
                                };
                            case 'debug[2]': // Data Interval
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 10000,
                                    outputRange: 1.0,
                                };
                            case 'debug[3]': // Execute Time
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 20,
                                    outputRange: 1.0,
                                };
                            case 'debug[4]': // Bus Busy Check
                            case 'debug[5]': // Read State Check
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 2,
                                    outputRange: 1.0,
                                };
                            case 'debug[6]': // Time since previous task uS
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 10000,
                                    outputRange: 1.0,
                                };
                            default:
                                return getCurveForMinMaxFields(fieldName);
                            }
```

**Comment by ctzsnooze** [comment]
Also in graph_config for GPS_DOP this gives consistent scaling of the DOP values:
```
                    case 'GPS_DOP':
                        switch (fieldName) {
                            case 'debug[0]': // Number of Satellites (now this is in normal GPS data, maybe gpsTrust?)
                            case 'debug[1]': // pDOP
                            case 'debug[2]': // hDOP
                            case 'debug[3]': // vDOP
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 200,
                                    outputRange: 1.0,
                                };
                            default:
                                return getCurveForMinMaxFields(fieldName);
                        }
```

**Review by ctzsnooze — DISMISSED** [review]

**Comment by ctzsnooze** [comment]
Looks good!

Unfortunately we missed out on the `GPS_UNIT_CONNECTION` debug, which goes just before `ATTITUDE`, and is used to debug a GPS module connection.

Details of what is needed is in [PR 645 ](https://github.com/betaflight/blackbox-log-viewer/pull/645/files).

PS... I have a feeling that the name of every debug is supposed to be in the big list in `DEBUG_MODE_COMPLETE` in `js/flightlog_fielddefs.js`

**Comment by ctzsnooze** [comment]
Please also use the angle mode graph scaling factors for `ANGLE_MODE` from [PR 620](https://github.com/betaflight/blackbox-log-viewer/pull/620/files), which are simply:
```
              case 'ANGLE_MODE':
                        switch (fieldName) {
                            case 'debug[0]': // angle target
                            case 'debug[3]': // angle achieved
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 1000,
                                    outputRange: 1.0,
                                };
                            case 'debug[1]': // angle error correction
                            case 'debug[2]':  // angle feedforward
                                return {
                                    offset: 0,
                                    power: 1.0,
                                    inputRange: 5000,
                                    outputRange: 1.0,
                                };
                            default:
                                return getCurveForMinMaxFields(fieldName);
                            }
     ```

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [inline js/flightlog_fields_presenter.js:1087]: This can be named 'Field Strength` I think.

File: js/flightlog_fields_presenter.js
Comment (ctzsnooze) [inline js/flightlog_fields_presenter.js:1091]: This can be called 'Lambda', it represents an estimate of the precision of the estimate of the bias values.

**Comment by ctzsnooze** [comment]
Looks really, really good now.  Fantastic!
Thank you very much @haslinghuis

**Pull request merged**
